src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimLipCallBackHandler.java

```diff
@@ -127,7 +129,26 @@ private void addOrginsationPoliciesforClaimantLip(CaseData.CaseDataBuilder caseD
                                                   .build());
         }
 
-        OrgPolicyUtils.addMissingOrgPolicies(caseDataBuilder);
+        if (caseData.getRespondent1OrganisationPolicy() == null) {
+            caseDataBuilder
+                .respondent1OrganisationPolicy(OrganisationPolicy.builder()
+                                                   .orgPolicyCaseAssignedRole(RESPONDENTSOLICITORONE.getFormattedName())
+                                                   .build());
+        }
+        if (caseData.getRespondent2OrganisationPolicy() == null) {
+            caseDataBuilder
+                .respondent2OrganisationPolicy(OrganisationPolicy.builder()
+                                                   .orgPolicyCaseAssignedRole(RESPONDENTSOLICITORTWO.getFormattedName())
+                                                   .build());
+        }
+
+        if (featureToggleService.isLipVLipEnabled() && caseData.isLipvLipOneVOne()) {
```

Comment:
No need to check the toggle as this handler is applicable for LiP v LiP claim only and will go live with R2.  I think We can simply set the respondent1OrganisationPolicy to '**DEFENDANT**'   role directly.